### DISCUSSION TITLE
breaking after upgrading to 0.11.0 from 0.4.3

### DISCUSSION THREAD

**Discussion by mritunjayz** [excalidraw/excalidraw#4919] [Q&A] [answered]
have we removed `getElementMap` function from npm package? getting error `getElementMap is not a function` after updating to 0.11.0
 cc: @ad1992

**Comment by ad1992** [comment, marked as answer]
Yes its removed in `0.11.0`, please take a look the the breaking changes when upgrading https://github.com/excalidraw/excalidraw/blob/master/src/packages/excalidraw/CHANGELOG.md#breaking-change
